app/controllers/advisors_controller.rb

```diff
@@ -0,0 +1,36 @@
+class AdvisorsController < ApplicationController
+
+  before_action :authenticate_advisor!
+  before_action :require_admin
+
+  def index
+    @advisors = Advisor.all
+  end
+
+  def update
```

Comment:
Advisors path should become an 'advisors' state. The update and destroy actions can be handled with restangular